Lib/test/test_print.py

```diff
@@ -155,7 +155,7 @@ def test_string_with_excessive_whitespace(self):
 
         self.assertIn('print("Hello World", end=" ")', str(context.exception))
 
-    def test_string_with_stream_redirection(self):
+    def test_print_string_with_stream_redirection(self):
         import sys
         python2_print_str = 'print >> sys.stderr, "message"'
```

Comment:
This is legal Python 3 syntax, so there's no need to use `exec` for these test cases - you can just put the expression in the body of the with statement.